### PR TITLE
AC-5: Integrate all new awards forms and update existing awards and their schemas

### PR DESCRIPTION
**Pull request by gaur-abhi524**
## Proposed changes

All the previous forms for applying for convocation medals were changed and new models were made for every award with new and updated schema

## Types of changes

_Put an `x` in the boxes that apply_

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [x] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [x] I have performed a self-review of my own code
-   [x] I have created new branch for this pull request
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [x] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

**Comment by Abhishek-PratapSingh** [comment]
Approve @akshatnema

File: FusionIIIT/templates/scholarshipsModule/forms/dandm_gold_medal.html [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+                <!--     <label>Brief Description:</label> -->
+                <!--     <p>This includes the following. <br/> -->
+                <!--         (a) Give the details of the list of papers you have studied for doing the project.<br/> -->
+                <!--         (b) Whether any publication, conference proceeding, conference paper presentation etc? If -->
+                <!--         yes, give proper details<br/> -->
+                <!--         (c) If some experimental setup, show the estimated cost:<br/> -->
+                <!--         (d) If some theoretical model development, discuss in brief:<br/> -->
+                <!--         (e) If some simulation results, what software/s did you use? <br/><br/> -->
+                <!--     </p> -->
+                <!-- </div> -->
```
Comment (akshatnema) [inline FusionIIIT/templates/scholarshipsModule/forms/dandm_gold_medal.html]: remove this comment

File: FusionIIIT/templates/scholarshipsModule/scholarships_student.html [hunk 2]
```diff
@@ -4,265 +4,254 @@
...
+            </a>
+            <!--Arihant Kumar Jain UI Card removed-->
+            <!--<a class="active item" data-tab="first">
+Previous Award Winners
+<i class="right floated chevron right icon"></i>
+</a>
+<a class="item" data-tab="two">
+Spacs Members Details
+<i class="right floated chevron right icon"></i>
+</a>-->
```
Comment (akshatnema) [inline FusionIIIT/templates/scholarshipsModule/scholarships_student.html]: Remove this comment

File: FusionIIIT/templates/scholarshipsModule/scholarships_student.html [hunk 3]
```diff
@@ -4,265 +4,254 @@
...
-                </a>-->
-                <a class="{% if last_clicked == 'get_MCM_Flag' or last_clicked == 'getConvocationFlag' or last_clicked == 'Submit_MCM' or last_clicked == 'Submit_Silver' or last_clicked == 'Submit_Gold' or last_clicked == 'Submit_dm' or last_clicked == 'studentApproveSubmit_MCM' or last_clicked == 'studentApproveSubmit_Con' %}active item {% else %}item {% endif %}" data-tab="two">
-                    Apply for Awards
-                    <i class="right floated chevron right icon"></i>
-                </a>
+        <!-- <div class="ui tab segment" data-tab="four">
+{% block viewmcm %}
+{% include 'scholarshipsModule/viewstatus.html' with winners=winners mcm=mcm gold=gold silver=silver dandm=dandm %}
+{% endblock %}
+</div>-->
```
Comment (akshatnema) [inline FusionIIIT/templates/scholarshipsModule/scholarships_student.html]: Remove this comment.

**Review by akshatnema — CHANGES_REQUESTED** [review]
Remove these type of comments from the PR.

**Comment by akshatnema** [comment]
Review required from @PranshuNayak and @samay-rgb.

**Review by PranshuNayak — COMMENTED** [review]
@gaur-abhi524 please remove indentation changes from models.py .

**Comment by gaur-abhi524** [comment]
@PranshuNayak can you mention the parts that need to be changed

**Comment by PranshuNayak** [comment]
@gaur-abhi524 just make sure that the models which have been changed or added are only highlighted so that we can easily review them.

**Review by PranshuNayak — COMMENTED** [review]
@gaur-abhi524 there are duplicate migrations in 0002 and 0003. Please push a single migration file for your changes , and don't modify the initial 0001

**Review by PranshuNayak — APPROVED** [review]
@gaur-abhi524 I have found migrations against 4 models.
newly created tables - DM_Proficiency_gold, IIITDM_Proficiency
modified tables - Director Gold, Director Silver

Since the file size is too large it's not possible to validate the migrations against each field, please cross-check at your end and verify.

Also only merging this PR will not create the database tables on 214 it needs to be deployed, so wait for some time if the table is not visible on 214.

**Review by samay-rgb — APPROVED** [review]

File: FusionIIIT/templates/scholarshipsModule/scholarships_student.html (hunk 2 above)
Comment (akshatnema) [inline FusionIIIT/templates/scholarshipsModule/scholarships_student.html]:
```suggestion

```

File: FusionIIIT/templates/scholarshipsModule/scholarships_student.html [hunk 4]
```diff
@@ -4,265 +4,254 @@
...
+            </a>
+
+            <!--<a class="item" data-tab="four">
+View Application Status
+<i class="right floated chevron right icon"></i>
+</a>
+<a class="item" data-tab="five">
+Browse Award Catalogue
+<i class="right floated chevron right icon"></i>
+</a>-->
```
Comment (akshatnema) [inline FusionIIIT/templates/scholarshipsModule/scholarships_student.html]:
```suggestion

```

File: FusionIIIT/templates/scholarshipsModule/scholarships_student.html [hunk 5]
```diff
@@ -4,265 +4,254 @@
...
 
+{% comment %}The Publications notionalprize ends here!{% endcomment %}
 
+{% comment %}The Publications about spacs  starts here!{% endcomment %}
+<div class="ui tab segment" data-tab="two">
+{% block aboutspacs %}
+{% include 'scholarshipsModule/aboutspacs.html' with hd=hd hd1=hd1 %}
+{% endblock %}
+</div>
+{% comment %}The Publications about spacs  ends here!{% endcomment %}
```
Comment (akshatnema) [inline FusionIIIT/templates/scholarshipsModule/scholarships_student.html]:
```suggestion

```

File: FusionIIIT/templates/scholarshipsModule/scholarships_student.html (hunk 3 above)
Comment (akshatnema) [inline FusionIIIT/templates/scholarshipsModule/scholarships_student.html]:
```suggestion
```

File: FusionIIIT/templates/scholarshipsModule/forms/dandm_gold_medal.html (hunk 1 above)
Comment (akshatnema) [inline FusionIIIT/templates/scholarshipsModule/forms/dandm_gold_medal.html]:
```suggestion
```

**Review by akshatnema — CHANGES_REQUESTED** [review]

**Review by akshatnema — APPROVED** [review]

**Pull request merged**